internal/integration/harness.go

```diff
@@ -1,4 +1,4 @@
-//go:build integration
+//go:build integration || infrastructure
```

Comment:
TBH we can probably just get rid of these tags in the `integration` package, but this works too.

Reply:
This felt a bit 'safer', in that we don't want them accidentally getting compiled into a production binary.